Modules/Loadable/Markups/Widgets/Resources/UI/qSlicerSimpleMarkupsWidget.ui

```diff
@@ -51,6 +51,12 @@
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
+         <string>vtkMRMLMarkupsLineNode</string>
```

Comment:
It would be better to keep it as is. All the modules that use this currently assume that the default is markups fiducials only. It may cause problems if suddenly many other markups types show up.

Reply:
Ok I've picked off `ENH: Additional markups types supported in qSlicerSimpleMarkupsWidget` from this PR.

Do you know which specific modules currently use a qSlicerSimpleMarkupsWidget?

Reply:
Thank you. Dozens of modules in various extensions use the markups place widget. This widget is used instead of a plain node selector whenever the user needs to add markup points, because on the toolbar you cannot choose which markups you want to edit.